posthog/api/notebook.py

```diff
@@ -171,6 +232,23 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 queryset = queryset.filter(last_modified_at__gt=relative_date_parse(request.GET["date_from"]))
             elif key == "date_to":
                 queryset = queryset.filter(last_modified_at__lt=relative_date_parse(request.GET["date_to"]))
+            elif key == "contains":
+                contains = request.GET["contains"]
+                match_pairs = contains.replace(",", " ").split(" ")
+                # content is a JSONB field that has an array of objects under the key "content"
+                # each of those (should) have a "type" field
+                # and for recordings that type is "ph-recording"
+                # each of those objects can have attrs which is a dict with id for the recording
+                for match_pair in match_pairs:
+                    target, match = match_pair.split(":")
+                    if target == "recording":
+                        if match == "true":
+                            queryset = queryset.filter(content__content__contains=[{"type": "ph-recording"}])
+                        elif match == "false":
+                            queryset = queryset.exclude(content__content__contains=[{"type": "ph-recording"}])
+                        else:
+                            # it could be a recording id
+                            queryset = queryset.filter(content__content__contains=[{"attrs": {"id": match}}])
```

Comment:
If I understand correctly, this would match any node, not just the one `ph-recording` ones? I think thats fine if its the case, just worth noting so its maybe clear there are some limitations there.

Wondering if it already makes sense to support any type of "target", simply by replacing `ph-recording` with `ph-{target}` ?